daisy/resource.go

```diff
@@ -174,3 +175,25 @@ func resourceNameHelper(name string, w *Workflow, exactName bool) string {
 	}
 	return name
 }
+
+type regionalResourceCache struct {
```

Comment:
I think twoDResourceCache and oneDResourceCache (or simular) would be better as we use regionalResourceCache for zonal resources as well as regional ones